### PR TITLE
GIBCT School Locations - Tables SHOULD be responsive

### PR DESCRIPTION
**Pull request by zurbergram**
## Description
https://github.com/department-of-veterans-affairs/va.gov-team/issues/11055#issuecomment-696781887

Added scopes to `th`s
Swapped first `td` in a `tr` in the `tbody` to be a `th`

Refactor GI's ResponsiveTable

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/gi/components/ResponsiveTable.jsx [hunk 1]
```diff
@@ -4,42 +4,48 @@ import classNames from 'classnames';
...
-            extraClass = '';
+      <tr key={key} className={rowClassName} role="row">
+        {columns.map((field, index) => {
+          const cellName = field.toLowerCase().replaceAll(' ', '-');
+          if (index === 0) {
+            return (
+              <th
+                className={`${cellName}-cell`}
+                scope="row"
+                /* eslint-disable-next-line jsx-a11y/no-interactive-element-to-noninteractive-role */
```
[the remark this answers is not in the document]
Comment (zurbergram) [reply src/applications/gi/components/ResponsiveTable.jsx:34]: See https://github.com/department-of-veterans-affairs/vets-website/pull/14288#issuecomment-696408767

**Review by shawkey-daniel — APPROVED** [review]
LGTM

File: src/applications/gi/components/ResponsiveTable.jsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (rianfowler) [reply src/applications/gi/components/ResponsiveTable.jsx:30]: @1Copenut is this a one off need or should we make a ticket to re-evaluate this rule?

**Review by rianfowler — APPROVED** [review]

**Review by delli-gatti-michael — APPROVED** [review]

File: src/applications/gi/components/ResponsiveTable.jsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (1Copenut) [reply src/applications/gi/components/ResponsiveTable.jsx:27]: @rianfowler This is a one-off. We need the roll to retain semantics when the table is collapsed to one column at mobile sizes. `display: block` destroys table semantics. The roles restore them.

**Review by 1Copenut — COMMENTED** [review]
Late to the review, but wanted to add a 👍 for this change. The table retained its semantics on iOS Safari + VoiceOver, so markup is solid. HTML validates per HTML5 validator and no axe errors. Will test in staging with other screen readers, but I don't foresee any issues.

**Review by 1Copenut — COMMENTED** [review]
This works as expected on all screen readers, including iOS on mobile.

**Pull request merged**